From 421be6fca5cbe0d9dd3a10467529eed53528c258 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Thu, 21 Jun 2018 09:03:00 +0300
Subject: [PATCH 079/144] Makefile: drop mention of *.cfgtmp

Since commit f916757300 (imx: Create distinct pre-processed mkimage
config files), *.cfgtmp files are no longer generated. There is no need
to remove them on the 'clean' target anymore.

Rename the .gitignore glob to *.cfgout.

Cc: Trent Piepho <tpiepho@impinj.com>
Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
 .gitignore | 2 +-
 Makefile   | 3 +--
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/.gitignore b/.gitignore
index 6bb0adefa7..8d18d6f49b 100644
--- a/.gitignore
+++ b/.gitignore
@@ -8,7 +8,7 @@
 .*
 *.a
 *.bin
-*.cfgtmp
+*.cfgout
 *.dtb
 *.dtb.S
 *.elf
diff --git a/Makefile b/Makefile
index 3516671320..4b3023b259 100644
--- a/Makefile
+++ b/Makefile
@@ -1609,13 +1609,12 @@ PHONY += $(clean-dirs) clean archclean
 $(clean-dirs):
 	$(Q)$(MAKE) $(clean)=$(patsubst _clean_%,%,$@)
 
-# TODO: Do not use *.cfgtmp
 clean: $(clean-dirs)
 	$(call cmd,rmdirs)
 	$(call cmd,rmfiles)
 	@find $(if $(KBUILD_EXTMOD), $(KBUILD_EXTMOD), .) $(RCS_FIND_IGNORE) \
 		\( -name '*.[oas]' -o -name '*.ko' -o -name '.*.cmd' \
-		-o -name '*.ko.*' -o -name '*.su' -o -name '*.cfgtmp' \
+		-o -name '*.ko.*' -o -name '*.su' \
 		-o -name '.*.d' -o -name '.*.tmp' -o -name '*.mod.c' \
 		-o -name '*.lex.c' -o -name '*.tab.[ch]' \
 		-o -name '*.symtypes' -o -name 'modules.order' \
-- 
2.18.0

